ispcrt/ispcrt.cpp

```diff
@@ -182,23 +182,23 @@ void ISPCLaunch(void **handlePtr, void *f, void *data, int countx, int county, i
         return ispc_launch_fptr(handlePtr, f, data, countx, county, countz);
     }
     // TODO: this code can be called directly from ISPC code. Not sure how to exit error safely.
-    throw std::runtime_error("Missing ISPCLaunch symbol");
+    abort();
 }
 
 void *ISPCAlloc(void **handlePtr, int64_t size, int32_t alignment) {
     if (ispc_alloc_fptr) {
         return ispc_alloc_fptr(handlePtr, size, alignment);
     }
     // TODO: this code can be called directly from ISPC code. Not sure how to exit error safely.
-    throw std::runtime_error("Missing ISPCAlloc symbol");
+    abort();
 }
 
 void ISPCSync(void *handle) {
     if (ispc_sync_fptr) {
         return ispc_sync_fptr(handle);
     }
     // TODO: this code can be called directly from ISPC code. Not sure how to exit error safely.
-    throw std::runtime_error("Missing ISPCSync symbol");
+    abort();
```

Comment:
Should we write the missing symbol message that was removed to stderr before aborting?

Reply:
Yes, we should.